src/components/CardsContainer.tsx

```diff
@@ -11,6 +10,24 @@ import { POINT_CODES } from '@/app/host/[roomId]/page'
 
 export default function CardsContainer() {
 	const [sortedUsersPoints, setSortedUsersPoints] = useState<ListenerRes[]>([])
+	const containerRef = useRef<HTMLDivElement>(null)
+	const [containerWidth, setContainerWidth] = useState(0)
+
+	// container width is need for the animation of the cards
+	useEffect(() => {
```

Comment:
suggestion: This could be a custom hook, for example:

```js
useResize(() => {
  setContainerWidth(containerRef.current.getBoundingClientRect().width)
})
```

Reply:
Would the entire useEffect go into the custom hook? And when do you decide to make a custom hook vs leaving it as it is?

Reply:
Good question. If it feels like reusable code, then yes, custom hook.

Reply:
DONE